cluster/charts/crossplane/README.md

```diff
@@ -95,6 +95,7 @@ securityContextRBACManager:
 | `rbacManager.managementPolicy`| The extent to which the RBAC manager will manage permissions. `All` indicates to manage all Crossplane controller and user roles. `Basic` indicates to only manage Crossplane controller roles and the `crossplane-admin`, `crossplane-edit`, and `crossplane-view` user roles. | `All` |
 | `alpha.oam.enabled` | Deploy the `crossplane/oam-kubernetes-runtime` Helm chart | `false` |
 | `metrics.enabled` | Expose Crossplane and RBAC Manager metrics endpoint | `false` |
+| `metrics.monitor.enabled` | Create prometheus Service Monitor and Service for scraping the metrics endpoint (Requires Prometheus to be installed) | `false` |
```

Comment:
Does this require the Prometheus Operator specifically to be installed? Should we call that out explicitly, or is it pretty much implied these days? It's been a while since I've deployed Prometheus.

Reply:
@negz I believe it depends on `kube-prometheus` because the core os `ServiceMonitor` CRD must be present https://github.com/crossplane/crossplane/pull/2069#discussion_r555168499